packages/wallet/src/TxBuilder/OutputBuilder.ts

```diff
@@ -0,0 +1,106 @@
+import { Cardano } from '@cardano-sdk/core';
+
+import {
+  DeepPartial,
+  MaybeValidTxOut,
+  OutputBuilder,
+  OutputValidationMinimumCoin,
+  OutputValidationMissingRequired,
+  OutputValidationTokenBundleSize
+} from './types';
+import { OutputValidation } from '../types';
+import { OutputValidator } from '../services';
+
+// Typescript is complaining about DeepPartial<Cardano.TokenMap> methods being optional
```

Comment:
Maybe best to not use DeepPartial then, but define a type that explicitly specifies  the Partial props